Desktop/components/JASP/Widgets/FileMenu/PrefsAdvanced.qml

```diff
@@ -333,9 +333,18 @@ ScrollView
 				to:					16
 				defaultValue:		4
 				stepSize:			1
+				KeyNavigation.tab:	showEnginesWindow
+				KeyNavigation.down:	showEnginesWindow
+				text:				qsTr("Maximum # of engines: ")
```

Comment:
I'd replace the `#` with "number".

Reply:
yeah you are right, I dont know why I was trying to keep that as short as possible there is quite some space.